data/core.yaml

```diff
@@ -630,6 +630,9 @@ en:
       improveOSM:
         tooltip: Missing data automatically detected by improveosm.org
         title: ImproveOSM Issues
+      osmose:
+        tooltip: Automatically detected map issues from osmose.openstreetmap.fr
```

Comment:
Hmm I know this is the same structure as the KeepRight description but both could use improvement. Maybe "Data issues detected by osmose.openstreetmap.fr"?